tools/probes/probes_main.c

```diff
@@ -153,6 +166,9 @@ void finalize_wave_files(struct wave_files *files)
 	/* and close all opened files */
 	/* check wave struct to understand the offsets */
 	for (i = 0; i < FILES_LIMIT; i++) {
+		if (!is_audio_format(files[i].fmt))
```

Comment:
interesting, files are ordered so that audio files come first?

Reply:
@lyakh Yikes, no, this is a bug, I need to skip to next one, not break.